### PR TITLE
[NO-TICKET] conditional font path for storybook styles

### PR DESCRIPTION
**Pull request by scheul93**
## Summary

In the Storybook webpack config, I created a conditional to set a different font path for design system styles. This is necessary because directory structures between dev & production builds are different (prod is flatter). This also resolves an issue with the new doc site where font files weren't loading.

## How to test

`yarn build-storybook:gatsby` to build storybook with conditional font path
`yarn build:gatsby && yarn serve:gatsby` to build doc site in production mode and serve it

Notice that before this change, you can see console errors about missing font files from storybook
Screen Shot 2022-05-13 at 11 53 02 AM

After change, no console errors and font files can be seen in network tab
Screen Shot 2022-05-13 at 11 47 34 AM

### CODE REVIEW

**Comment by scheul93** [comment]
@pwolfert, not sure if this resolves previous storybook build oddities you've encountered in the past

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -56,7 +67,7 @@ module.exports = async (webpackConfig) => {
         {
           loader: 'sass-loader',
           options: {
-            additionalData: `@import '${getStylesPath()}';`,
+            additionalData: `${getFontPath()} @import '${getStylesPath()}';`,
```
Comment (pwolfert) [inline .storybook/webpack.config.js:70]: Do we need to do the same thing for `./images` and `$image-path`?

**Review by pwolfert — APPROVED** [review]
🔡 🎉

**Comment by pwolfert** [comment]
> not sure if this resolves previous storybook build oddities you've encountered in the past

I think this fixes it! That was so simple!

File: .storybook/webpack.config.js (hunk 1 above)
Comment (scheul93) [reply .storybook/webpack.config.js]: Added, because it IS needed 🙂

**Pull request merged**